src/hooks/useTokenIdentityRegistry.ts

```diff
@@ -1,32 +1,43 @@
 import { useEffect, useState } from "react";
-import { EvmAddress, TokenNameItem } from "@/types/types";
+import { TokenNameItem, EvmAddress } from "@/types/types";
 import { WatchContractEventReturnType } from "@/services/contracts/watchContractEvent";
 import {
   readTokenIdentityRegistry,
   watchIdentityRegistryIdentityRegisteredEvent,
 } from "@/services/contracts/wagmiGenActions";
 
 export function useTokenIdentityRegistry(tokenSelected?: TokenNameItem) {
-  const [registry, setRegistry] = useState("");
-  const [registryAgents, setRegistryAgents] = useState<string[]>([]);
+  const [registry, setRegistry] = useState<EvmAddress>();
+  const [registryIdentities, setRegistryIdentities] = useState<
+    { identityAddr: EvmAddress; walletAddr: EvmAddress }[]
```

Comment:
In general, using full words would improve the code readability

identityAddr -> identityAddress
walletAddr -> walletAddress